package.json

```diff
@@ -30,7 +30,7 @@
     "ethereum-waffle": "^3.4.0",
     "ethers": "^5.5.2",
     "fishery": "^2.0.0",
-    "hardhat": "^2.6.5",
+    "hardhat": "^2.9.1",
```

Comment:
this hardhat update is so I can use "--grep" command with hardhat test for convenience